clicommand/cancel_signal.go

```diff
@@ -0,0 +1,36 @@
+package clicommand
+
+import "github.com/urfave/cli"
+
+const (
+	defaultCancelGracePeriod = 10
+
+	// This will be increased to 9 in a future release of the agent.
+	defaultSignalGracePeriod = 0
+)
+
+var (
+	// cancel grace period must be stricly longer than signal grace period
+	_ uint = defaultCancelGracePeriod - defaultSignalGracePeriod - 1
```

Comment:
Heh, clever use of untyped consts.

Reply:
I was tossing up between this and making it a test. Because both sources of truth are in the same file, I think it's more convenient to get the feedback from the compiler. And if you have a language server, you get it directly in the editor.